Jim-Hodapp-Coaching/esp32-wroom-rp
# Adds examples/secrets.rs

## Description
Adds `examples/secrets.rs` to store ssid, passphrase for use in any example application.

#### GitHub Issue: N/A

### Changes
* Adds `examples/secrets.rs` which includes `SSID` and `PASSPHRASE` constants
* Uses `SSID`/`PASSPHRASE` in `joins.rs`

### Testing Strategy
1. Set your local SSID and passphrase in `examples/secrets.rs`
2. Run the `join` example application like `RUST_BACKTRACE=1 cargo run --example join`
3. Observe that that the example connects and then disconnects from your local WiFi network

### Concerns
None
